pages/Nix/Options & Overrides.md

```diff
@@ -29,6 +29,17 @@ programs.hyprland = { # or wayland.windowManager.hyprland
 };
 ```
 
+### HM module
```

Comment:
This should go below `## Options descriptions`, before or after `### XWayland`. It should be something like

    ### Plugins

    ```nix
    wayland.windowManager.hyprland = {
      plugins = [
        inputs.hyprland-plugins.packages.${pkgs.system}.hyprbars # package
        "/home/<user>/Projects/my-hyprland-plugin/libmy-hyprland-plugin.so" # path
      ];
    };
    ```

And then you should include more info about how to build plugins, like "look at how pacakges are defined in `hyprland-packages`".

Reply:
Alright

Reply:
Done. You probably want to squash this.